code/modules/mailorder/modr_entries.dm

```diff
@@ -0,0 +1,365 @@
+//information for catalogue sale entries
+ABSTRACT_TYPE(/datum/mail_order)
+/datum/mail_order
+	var/name = "Juicer Schweet's Spaghetti Western"
+	var/cleanname = null //sanitary version of name created for references
+	var/desc = "Be like the three-second elephant with heated value in space-bark."
+	var/list/order_items = list() // should not exceed 7 items, as mail order is sent in a box item
+	var/cost = PAY_UNTRAINED
+	var/list/order_perm = list() // optional access requirement to order a given item
+
+	New()
+		..()
+		src.cleanname = ckey(src.name)
+
+//Tanhony & Sons: journalistic equipment, primarily AV
+ABSTRACT_TYPE(/datum/mail_order/audiovideo)
+/datum/mail_order/audiovideo
+
+	audiolog
+		name = "Audio Log"
+		desc = "A must-have for catching that crucial conversation."
+		order_items = list(/obj/item/device/audio_log)
+		cost = PAY_TRADESMAN
+
+	audiotape
+		name = "Audio Tape"
+		desc = "Audio storage tape for use in suitable audio devices."
+		order_items = list(/obj/item/audio_tape)
+		cost = PAY_TRADESMAN / 3
+
+	laptop
+		name = "Personal Laptop"
+		desc = "The latest in micro-computing, right at your fingertips."
+		order_items = list(/obj/item/luggable_computer/personal)
+		cost = PAY_EXECUTIVE * 2
+
+	floppy
+		name = "Blank Computer Data Disk"
+		desc = "Removable data storage, often used in ThinkDOS computers."
+		order_items = list(/obj/item/disk/data/floppy)
+		cost = PAY_TRADESMAN / 3
+
+	photocam
+		name = "Handheld Photo Camera"
+		desc = "Captures high-fidelity images with insta-develop film."
+		order_items = list(/obj/item/camera)
+		cost = PAY_TRADESMAN
+
+	photofilm
+		name = "Photo Film Cartridge"
+		desc = "A replacement film cartridge for handheld photo cameras."
+		order_items = list(/obj/item/camera_film)
+		cost = PAY_TRADESMAN / 3
```

Comment:
Changes the balance of spy-thief given how accessible this would make the "photo of" objectives. Worth discussion.

Reply:
Cameras are a neat little thing to have greater access to but they do seem pretty rare (tourists, spies, centcomm, forensics locker, and random spawners)

If scarcity of film is supposed to drive spy vs spy interaction this does seem like a bummer.

Reply:
Could ramp up the price a bit? I'm rather fond of having cameras as a purchaseable item, but not quite sure what to do about the spy thief ramifications.